### PR TITLE
Switch to rate-limiter-flexible

### PR DESCRIPTION
**Pull request by Page-**

### CODE REVIEW

File: src/lib/rate-limiting.ts [hunk 1]
```diff
@@ -54,29 +67,20 @@ const getStore = _.once(() => {
...
-) => {
-	console.error('Blocked by rate limiting: ' + req.originalUrl);
-	ExpressBrute.FailTooManyRequests(req, res, next, nextValidRequestDate);
+	// const opts: ExpressBruteRedisOpts = {
+	// 	client,
+	// 	prefix: 'api:ratelimiting:',
+	// };
+
+	// const redisStore = new ExpressBruteRedis(opts);
+	return new RateLimiterRedis({
```
Comment (afitzek) [inline src/lib/rate-limiting.ts]: Is there a way to have a key prefix for the redis backend as well? We should try to namespace redis.

**Comment by afitzek** [comment]
@balena-ci retest

**Comment by Page-** [comment]
@balena-ci retest

**Comment by Page-** [comment]
@balena-ci retest

**Pull request merged**